tests/e2e/helpers.bash

```diff
@@ -139,6 +139,28 @@ bria_lnd_init() {
   echo "Bria Initialization Complete"
 }
 
+bria_multisig_init() {
```

Comment:
this should not be in helpers. it is the critical testing path!
Please reuse `bria_init` and do anything else needed in the test.